package.json

```diff
@@ -40,6 +40,7 @@
     "babel": "^5.8.25",
     "babel-core": "^5.8.25",
     "babel-eslint": "^4.1.1",
+    "babel-relay-plugin": "0.7.1",
```

Comment:
I found that travis-ci will not green light the PR in npm3 since it no longer looks at peer dependencies, so I threw it in devDeps